### ISSUE TITLE
Problem with password/seed after 1st time installation of Electrum

### ISSUE THREAD

**Issue by Whatsup2000** [spesmilo/electrum#2155]
OK, I feel totally frustrated right now..

I've installed Electrum 2.7.12 completely without any problems. It didn't ask me for a password; it didn't show me any seed either. I am absolutely sure of this. 

Next, I was able to RECEIVE 0.2 BTC on my account without any problems. However, now I want to SEND it to someone, it suddenly asks me for a password.. Of which I am ABSOLUTELY, POSITIVELY sure that I was never asked to provide. Since I've never seen any 'seed' neither, I am probably unable to access my money ever again. EVER!

Entering a blank password doesn't work by the way. 

Am I missing something here? Or am I just f***ed??

Regards.
B.

**Comment by JazzTp** [comment]
Hi, I'm a newbie myself, so don't take my words as coming from en expert
user.

I bet developers will need more info, like operative system and version,
and wallet type.

In my experience, Electrum _always_ asked very clearly everything at
startup in case it can't find a previously created wallet in the place of
the default wallet, or you haven't included one as a command line parameter.
(In case you open the watching only part of a wallet, it also warns about
it.)

The only thing I can think of (possibly nothing to do with the real
situation) is that on your system there might have been a previously
created wallet, a "full" one not watching only (notice that you normally
want to keep the "full' one which includes the private key OFFLINE as a
cold storage, out of reach of any intruder), and that Electrum might have
found it in the location of the default one.

Had you tried Electrum before, had you installed as previous version and
created a wallet? I don't know if reinstalling Electrum erases any previous
default wallet or lets it survive, on your operative system.

After creating a new wallet, even if everything went as expected and I had
noted and verified a seed, and I had noted and double typed a password, I
ALWAYS tested the wallet with a very small sum and only AFTER verifying
that I could spend coins FROM the wallet I eventually sent any non trivial
amount into it.

OK, I feel totally frustrated right now..

I've installed Electrum 2.7.12 completely without any problems. It didn't
ask me for a password; it didn't show me any seed either. I am absolutely
sure of this.

Next, I was able to RECEIVE 0.2 BTC on my account without any problems.
However, now I want to SEND it to someone, it suddenly asks me for a
password.. Of which I am ABSOLUTELY, POSITIVELY sure that I was never asked
to provide. Since I've never seen any 'seed' neither, I am probably unable
to access my money ever again. EVER!

Entering a blank password doesn't work by the way.

Am I missing something here? Or am I just f***ed??

Regards.
B.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/spesmilo/electrum/issues/2155>, or mute the thread
